homeassistant/components/miele/strings.json

```
@@ -1096,6 +1099,28 @@
           "name": "Program ID"
         }
       }
+    },
+    "set_program_oven": {
+      "name": "Set program on oven",
+      "description": "Sets and starts a program on the appliance.",
```

If you want to keep this identical to the generic `set_program` action a reference will do it, too:

```suggestion
      "description": "[%key:component::miele::services::set_program::description%]",
```